### PR TITLE
docs(readme): add license notice and additional info

### PR DESCRIPTION
**Pull request by owlot**
I compared the README.md with our [Template README.md](https://github.com/Arrow-air/svc-template-rust/blob/develop/README.md) and added missing blocks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,6 +107,10 @@ Learn more about us:
 - [Arrow Docs](https://www.arrowair.com/docs/intro)
 - [Discord](https://discord.com/invite/arrow)
 
+## LICENSE Notice
+
+Please note that svc-template is under BUSL license until the Change Date, currently the earlier of two years from the release date. Exceptions to the license may be specified by Arrow Governance via Additional Use Grants, which can, for example, allow svc-template to be deployed for certain production uses. Please reach out to Arrow DAO to request a DAO vote for exceptions to the license, or to move up the Change Date.
```
Comment (GoodluckH) [inline README.md:112]: Should be svc-scheduler?

**Review by GoodluckH — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (owlot) [reply README.md]: fixed

**Review by GoodluckH — COMMENTED** [review]
We are missing two badges for this repo. I think for other repos, there's the new release badge and the sanity check badge.

**Review by GoodluckH — APPROVED** [review]

**Comment by ServiceDog** [comment]
Sanity and release badges added, signing off

**Review by ServiceDog — APPROVED** [review]

**Pull request merged**